core-blocks/embed/index.js

```diff
@@ -30,7 +30,7 @@ const HOSTS_NO_PREVIEWS = [ 'facebook.com' ];
 
 function getEmbedBlockSettings( { title, description, icon, category = 'embed', transforms, keywords = [] } ) {
 	// translators: %s: Name of service (e.g. VideoPress, YouTube)
-	const blockDescription = description || sprintf( __( 'Paste URLs from %s to embed the content in this block' ), title );
+	const blockDescription = description || sprintf( __( 'Add a block that displays content pulled from other sites, like Twitter, Instagram or YouTube.' ), title );
```

Comment:
As of this change, there's no placeholder content in the string, but we still use placeholder substitution (`sprintf`) and include a translator comment describing the placeholder on the preceding line.

Both of these should have been removed.